src/bin/ipa_bench/gen_events.rs

```diff
@@ -105,7 +154,8 @@ pub fn generate_events<R: RngCore + CryptoRng, W: io::Write>(
 
         for _ in 0..reach {
             // # of devices == # of matchkeys
-            let devices = sample.devices_per_user(rng);
+            // Hard code this number to 1 as we only have a solution for one match key.
+            let devices = 1;
```

Comment:
This is a misunderstanding. Just one match key can certainly be useful across multiple devices. If the same (single) match key is set to the same value on multiple devices - the events can still be joined together.

Multiple match keys just gives a "boost" on match rates - that's all.

Reply:
Agree. I should've said "at the moment, we only have ...".